modules/react-components/src/components/error/error-boundary.tsx

```diff
@@ -31,6 +31,7 @@ interface ErrorBoundaryState {
  */
 interface ErrorBoundaryProps {
     fallback: React.ReactNode;
+    onChunkLoadError: (state: boolean) => void;
```

Comment:
We do not need the `state` here right?
```suggestion
    onChunkLoadError: () => void;
```